.editorconfig

```diff
@@ -0,0 +1,12 @@
+; editor configuration powered by http://editorconfig.org/
+; Top-most EditorConfig file
+root = true
+
+; Windows newlines
+[*]
+end_of_line = crlf
+indent_style = space
+indent_size = 4
```

Comment:
I'd add
```editorconfig
charset = utf-8
insert_final_newline = true
```

Reply:
Good point, PowerShell tends to do UTF-16 when using echo (been burnt by that)